tests/testthat/test-vivax.R

```diff
@@ -24,3 +24,41 @@ test_that('Test difference between falciparum and vivax parameter lists', {
   expect_true(all(names(vivax_parameters[!names(vivax_parameters) %in% names(falciparum_parameters)]) %in%
                     c("dpcr_max","dpcr_min","kpcr","apcr50","init_iaa","init_iam","ra","ua","b","philm_min","philm_max","klm","alm50","ca","f","gammal","init_hyp","kmax")))
 })
+
+test_that('Test age structure should not change vivax infectivity', {
```

Comment:
I think I understand what you're getting at with this test, but could it be done more simply just by setting the `variables$id` value for your pop, then checking what `new_infectivity` has been set? I think that would should highlight that Pf depends on the variable and Pv doesn't?

Reply:
This one I find a bit trickier. The falciparum code has a function to calculate asymptomatic_infectivity (where you could set id), but the vivax doesn't have a comparable function because it's constant. The options for outer functions to run that generate infectivity from variables are in create_variables, recovery_outcome_process and schedule_infections, but all these are quite bulky - there isn't really a neat way of checking immunity -> infectivity, as far as I can tell.